### PR TITLE
NodeJs week1 homework

### PR DESCRIPTION
**Pull request by sayali05**

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -21,6 +24,76 @@ app.use(cors());
...
+
+//Respond with the first meal (meaning with the minimum id)
+app.get("/first-meal", async (req, res) => {
+  try {
+    const meal = await knex
+      .select("title", "description")
+      .from("meal")
+      .orderBy("id")
+      .first();
+    res.status(200).json(meal);
```
Comment (raihanmehran) [inline src/backend/app.js:77]: add an `if` condition for the response. If meal doesn't exist it should respond with a `404` response with a message, that the meal not found or something. And if exist it should return meal.

File: src/backend/app.js [hunk 2]
```diff
@@ -21,6 +24,76 @@ app.use(cors());
...
+});
+//Respond with the last meal (meaning with the maximum id)
+app.get("/last-meal", async (req, res) => {
+  try {
+    const meal = await knex
+      .select("title", "description")
+      .from("meal")
+      .orderBy("id", "desc")
+      .limit(1);
+    res.status(200).json(meal);
```
Comment (raihanmehran) [inline src/backend/app.js:90]: add an `if` condition for the response. If meal doesn't exist it should respond with a `404` response with a message, that the meal not found or something. And if exist it should return meal.

**Review by raihanmehran — COMMENTED** [review]
Nice Solution 👍 

Happy Coding :)

File: src/backend/app.js (hunk 2 above)
Comment (sayali05) [reply src/backend/app.js:90]: Thank you for the review.. I will update code as per suggestion.

**Pull request merged**